src/main/resources/META-INF/rewrite/jakarta-faces-4.yml

```diff
@@ -39,6 +40,16 @@ recipeList:
   - org.openrewrite.java.migrate.jakarta.UpgradeFacesOpenSourceLibraries
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.UpdateJakartaFacesApi4
+displayName: Update Jakarta EE Java Faces Dependencies to 4.0.x.
+description: Update Jakarta EE Java Faces Dependencies to 4.0.x.
+recipeList:
+  - org.openrewrite.java.dependencies.UpgradeDependencyVersion:
+      groupId: jakarta.faces
+      artifactId: jakarta.faces-api
+      newVersion: 4.0.x
```

Comment:
needed to add this, previously the EE9 migration included the Faces 4 (part of EE10) update